app/src/main/java/com/example/keniphhomemaintenance/Navigation.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.keniphhomemaintenance
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.example.keniphhomemaintenance.dwellings.DwellingsScreenView
+import com.example.keniphhomemaintenance.ui.Screen
+
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    val defaultDwellingName = stringResource(R.string.default_dwelling_name)
```

Comment:
Would prefer these just be used directly in the arguments; these variables are only ever used once so I don't thing we even need to have them

Reply:
If I put stringResource(R.string.default_dwelling_name) right into the composable I get the error:

@Composable invocations can only happen from the context of a @Composable function

Reply:
(discussion on fb messenger) icky variable